text/0000-intro-video-script.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Intro video script
+- Start Date: 2021-05-18
+- Author(s): yaelharel
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Improving documentation is a project priority for 2021.
+This RFC is an attempt to create an easy and clear starting point for new users and contributors.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+It is currently relatively challenging for new users and contributors to understand what buildpacks are and what is the purpose of our project.
+It can be very useful to have a ~1 minute introductory video to provide high-level answers to these questions.
+We can post this video on our website and share it on social media.
+Hopefully this will help familiarize more people with our project.
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+### The script
+
+As an application developer, you should only focus on what you do best - writing code, without having to worry about image security, creating minimal images, and your container build strategy in general. But how much time have you wasted struggled to wrangle yet another Dockerfile? Copy-pasting random Dockerfile snippets in every project? Buildpacks can help you with just that! They are a better approach to building container images for applications.
+
+Buildpacks are programs that transform source code into container images by analyzing the code and determining the best way to build it. They can provide dependencies, configuration, run compilation, and much more! A buildpack can be written by anyone, and shared with everyone! Google, Heroku, Salesforce, and the Paketo project provide buildpacks for a wide variety of language families, and you can discover even more by searching the buildpack registry. With buildpacks, you can use best-in-class container build strategies from industry experts with ease.
+
+Even more so, the Cloud Native Buildpacks project provides a specification for buildpack authors and platforms that ensures builds are fast, secure, customizable, and can run on any cloud. It is a Cloud Native Computing Foundation project with many existing integrations and many more to come. Once applications are in containers there is an incredible community of tools and products that can help get your app online and support its growth. Heroku, Hashicorp Waypoint, VMWare Tanzu, Spaceship, CircleCI, and Google Cloud Run all support using buildpacks to make container images.
```

Comment:
I'm not sure I'm keen on name dropping just a few integrations or platforms. For buildpack providers it doesn't seem like a huge deal since we already "suggest" them via things like `pack builders suggest`. For platforms and integrations, I'm curious as to why only this subset was selected.

Concerns:

* It seems subjectively bias to only call out a subset of integrations/platforms.
* It increases the staleness factor as platforms may come and go.

That said, I tried to identify the integrations as platforms and alphabetize them.

```suggestion
Even more so, the Cloud Native Buildpacks project provides a specification for buildpack authors and platforms that ensures builds are fast, secure, customizable, and can run on any cloud. It is a Cloud Native Computing Foundation project with many existing integrations. Platforms such as Google Cloud Run, Hashicorp Waypoint, Heroku, Spaceship, and VMWare Tanzu, just to name a few, all support using buildpacks to make container images.
```

Reply:
I'm going to make this change but I have to say that I'm not sure the last sentence (starting with "Platforms such as...") is necessary.
Any thoughts?